events/events-team/content/meet-and-greet.md

```diff
@@ -0,0 +1,94 @@
+
+If Kubernetes has a new SIG this year, or a team that really wants to recruit, notify the staff as soon as possible so that they can order new gear for that group.
+
+### 1 Month Before: Recruit SIG staffers
+
+Prepare a sign-up sheet for SIG volunteers, usually a spreadsheet so that folks from SIGs can see who else has signed up for their SIG.  Then work with Contributor Comms in order to message all of the SIG leads.  Notify folks on the following channels:
+
+- Slack, #chairs-and-techleads
+- Slack, #contributor-summit
+- kubernetes-dev mailing list
+- LWKD
+- Ideally, a message to each of the primary #sig-* channels (this is a good task for a shadow, and should probably be bundled with outer communications)
+
+This solicitation might be bundled with other contributor-summit messaging that needs to go to all SIG leads, such as a call for booking SIG meeting rooms.
+
+Add a page in the Contributor Summit website for the Meet & Greet.
+
+# Follow-ups
+
+Check weekly to see who's signed up.  While participation by all SIGs is valuable, the following SIGs are particularly popular with 1st-time contributors, and if nobody signs up you should message them:  SIG-Docs, SIG-Release, SIG-CLI, SIG-Contributor Experience, SIG-K8s-Infra.
+
+### 1 Week Before: Solicit Attendees and Send Reminders
+
+By this time, CNCF events should have given you a room number and day. Send out a reminder to all of the chairs and techleads, and give them the updated information including the location and time.  Update the Summit website with the full information.
+
+Working with Comms, advertise the Meet & Greet to Kubecon attendees and community members.  This should include advertising it on community channels, including social media, #contributor-summit, #kubernetes-contributors, and kubernetes-dev ML, and LWKD.  You should also make sure that it gets into at least one of Kubecon's official messages to attendees; ask CNCF about this.
+
+Stretch Goal: prepare a post-M&G survey.  We haven't actually done one of these before, so you'll be starting from scratch.  Good questions to ask: how do they feel it went, how many folks from their SIG were there, and how many contributors did they talk to (and maybe a newbie/experienced estimate).  For attendees, it would ask which SIG(s) they were seeking, why, and if they found what they were looking for. Print QR codes for the survey to have on the tables.
+
+Decide whether you'll have a sign-in sheet for attendees.  This is unnecessary if you're doing a survey, but if you're not, you should at least have sign-ins. If so, design & print it out.
+
+### 1 Day Before
+
+If possible, check out the space assigned to the M&G, just in case there are problems with it.  If there are, raise them with our CNCF event staff.
```

Comment:
```suggestion
If possible, check out the space assigned to the M&G, just in case there are problems with it.  If there are, raise them with our CNCF event staff.

Check that all SIGs who intended to attend are accounted for and were able to make it to the event. In the case of anyone with last-minute travel cancellations, identify if someone can cover for them. If a SIG cannot get members out due to travel issues, solicit help to staff their table from related groups or SIG ContribEx.
```